spec/factories/async_transactions.rb

```diff
@@ -5,7 +5,7 @@
     sequence(:user_uuid, 100) { |n| "abcd789-6af0-45e1-a9e2-394347a#{n}" }
     sequence(:source_id, &:to_s)
     source              'vet360'
-    status              'started'
+    status              'requested'
```

Comment:
Perhaps reference the constant here from the other class (where we defined `REQUESTED`), instead of using a string?